core/src/main/java/bio/terra/pearl/core/model/participant/Enrollee.java

```diff
@@ -35,6 +35,8 @@ public class Enrollee extends BaseEntity {
     private boolean subject = true; // whether this Enrollee is a primary subject of the study (as opposed to just a proxy or family member)
     private boolean consented;
 
+    @Builder.Default
+    private List<FamilyEnrollee> familyEnrollees = new ArrayList<>();
```

Comment:
I think it muddles the data model representation to have both Families and FamilyEnrollees on Enrollee.  How about we just have FamilyEnrollees on Enrollee, and then update FamilyEnrollee to have a Family object.  That matches how we handle, e.g. StudyEnvironment -> StudyEnvironmentSurvey -> Survey.